frontend/src/metabase-lib/Question.ts

```diff
@@ -1074,19 +1074,7 @@ class QuestionInner {
   } = {}): Promise<[Dataset]> {
     // TODO Atte Keinänen 7/5/17: Should we clean this query with Query.cleanQuery(query) before executing it?
     const canUseCardApiEndpoint = !isDirty && this.isSaved();
-    const parameters = this.parameters()
-      // include only parameters that have a value applied
-      .filter(param => _.has(param, "value"))
-      // only the superset of parameters object that API expects
-      .map(param => _.pick(param, "type", "target", "value", "id"))
```

Comment:
This line is missing from https://github.com/metabase/metabase/pull/27005/files#diff-8b5188e613c51e0dd17810bc8859b25544494e8b319ecdb428038048fe4b0542R25.

Not sure if that's intended. Since I'm not quite familiar with the code here, but the diff looks questionable to me.